src/io/flutter/logging/FlutterErrorHelper.java

```diff
@@ -13,7 +13,7 @@
 import java.util.regex.Pattern;
 
 public class FlutterErrorHelper {
-  private static final Pattern numberPattern = Pattern.compile("[0-9]+");
+  private static final Pattern numberPattern = Pattern.compile("[0-9][0-9.]*");
```

Comment:
nit: 
`[0-9]+[.][0-9]*`
would be a little more precise and avoid catching 10.1.0.33

Reply:
updated to `[0-9]+(\\.[0-9]+)?`